play/PlayerRules/CharacterCreation/05_ArmourList.txt

```diff
@@ -88,12 +71,67 @@ this by providing 'Hard Points'. Hard Points are special joints and fittings tha
 make it easy to attach equipment. When you purchase equipment, it will list how
 many Hard Points it consumes. You can't attach more equipment than you have available
 Hard Point space.
+	Hard Point Equipment/Shields/Cloaking often come in two sizes, "Primary Modules" and 
+"Secondary Modules". After you purchase and install a Primary Module, you can also purchase
+and install a Secondary Module(s) that augments the benefits of the Primary Module. The 
+benefits are listed in the description of a given module. They do not mix and match,
+Secondary Modules only work for the Primary Module they are designed for.

```

Comment:
I like this description of primary and secondary modules. I didn't know this was a thing, but after reading like 4 sentences, I feel like I get it.